apps/ios_rpc/README.md

```diff
@@ -103,7 +40,13 @@ This custom `dlopen` mechanic is integrated into TVM RPC as plugin and registere
 application.
 
 The custom implementation of `dlopen` and other functions from `dlfcn.h` header are placed in separate repository,
-and will be downloaded automatically during cmake build for iOS. To run cmake build you may use next flags:
+and will be downloaded automatically during cmake build for iOS.
+
+Also, it is necessary to build `tvm_runtime.dylib` for our iOS device. The iOS
+TVM RPC application will be linked with this library. Use the following cmake
+flags:
```

Comment:
missing cmake flags?

Reply:
The cmake flags are specified below. Removed this sentence.